modules/lss-missionKeyword/missionKeyword.user.js

```diff
@@ -1282,189 +1585,194 @@
     var color_thw      = '#03a8f9';
     var color_sonst    = '#03a8f9';
 
-	//Stichwort Element
-    var title_stichwort = document.getElementById('missionH1');
+	if(I18n.locale == 'de')
+		default_aao = default_aao_de;
+	else if(I18n.locale == 'en')
+		default_aao = default_aao_en;
+	else if(I18n.locale == 'nl')
+		default_aao = default_aao_nl;
+	else
+		default_aao = default_aao_de;
+		
 	
-	//Anzahl Patientenbalken
-    var anzahl_patients = document.getElementsByClassName("patient_progress").length;
-    var aao_text = '';
+	$.each(default_aao, function(key, val) {
+		var settingsPropertyName = 'missionKeyword-'+key;
+		var tmpObject = {
+			[settingsPropertyName] : {
+				"default": val,
+				"ui": {
+					"label": I18n.t('lssm.missionKeywordDefaults.'+key)+ " (ID "+key+")",
+					"type": "text",
+					"description": ""
+				}
+			}
+		};
+		
+		$.extend(managedSettings.settings, tmpObject);
+		
+	});
+		
+	lssm.managedSettings.register(managedSettings);	
 	
-	//Link für Hilfe (wird für ID benötigt)
-    var help_el = document.getElementById('mission_help');
-    var help = "";
-    if (help_el !== null)
-        help = help_el.href;
-
-    var tmp_arr = help.split('/');
-    var length = tmp_arr.length;
+	function getSetting(setting) {
+		return lssm.managedSettings.getSetting(LSS_MISSIONKEYWORD_STORAGE, setting);
+	}
+	
+	function missionKeyword() {
```

Comment:
Hier könntest du z.B. die verschiedenen Sprachanpassungen raus nehmen und diese in je eine Funktion schieben. Sodass du nur noch für die Sprachversion die Funktion aufrufen musst. (ich meine das in deinen if else Geschichten )